scripts/combine/setupHighPU_Wmass_mT.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python3
```

Comment:
I guess we probably want to either integrate this with the pt_eta script or at least refactor the code to reuse it. I can take a look at doing this. Also going to look at refactoring things so uncertainties on mT can be added more easily